lusSTR/filter_settings.py

```diff
@@ -370,9 +445,9 @@ def flags(allele_df):
 def same_size_filter(df, metadata):
     final_df = pd.DataFrame()
     al_list = df["RU_Allele"].unique()
-    for ru_allele in al_list:
+    for ce_allele in al_list:
         df_filt = (
-            df[df["RU_Allele"] == ru_allele]
+            df[df["RU_Allele"] == ce_allele]
```

Comment:
Trying to be consistent between "CE Allele" and "RU Allele". Since STRmix outputs it as "CE Allele", I'm trying to stick with that. However, lusSTR output reports it as "RU Allele". Thoughts on changing all of lusSTR? I can if you think that's worth the effort.

Reply:
I just went ahead and changed all of lusSTR to be consistent.